src/main/java/domain/calculator/Calculator.java

```diff
@@ -0,0 +1,38 @@
+package domain.calculator;
+
+import java.util.Arrays;
+
+public class Calculator {
+
+    public String escape(String numbers) {
+        String custom = "";
+        if (numbers.contains("//") && numbers.contains("\n")) {
+            custom = numbers.substring(2, 3);
+            return custom;
+        }
+        return custom;
+    }
+
+    public boolean isNullAndSpace(String numbers) {
+        if (numbers.length() == 0 || numbers == null) {
+            return true;
+        }
+        return false;
+    }
+
+    public int add(String numbers) {
+        if (isNullAndSpace(numbers)) {
+            return 0;
+        }
+        String custom = escape(numbers);
+
+        int sum = Arrays.stream(numbers.split("[,|:|//" + custom + "\\n]+"))
+                .filter((c) -> {
+                    if (c.matches("[^0-9][-0-9]+")) {
+                        throw new RuntimeException();
+                    }
+                    return !c.equals("");
+                }).mapToInt(Integer::parseInt).sum();
+        return sum;
```

Comment:
```suggestion
        return Arrays.stream(numbers.split("[,|:|//" + custom + "\\n]+"))
                .filter((c) -> {
                    if (c.matches("[^0-9][-0-9]+")) {
                        throw new RuntimeException();
                    }
                    return !c.equals("");
                }).mapToInt(Integer::parseInt).sum();
```
변수를 할당해서 다른 곳에서 사용하지 않으면 바로 반환해도 되겠죠?